easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -345,8 +466,10 @@ def sanity_check_step(self, *args, **kwargs):
         """
         Custom sanity check for Python packages
         """
-        if not 'exts_filter' in kwargs:
-            kwargs.update({'exts_filter': EXTS_FILTER_PYTHON_PACKAGES})
+        if 'exts_filter' not in kwargs:
+            orig_exts_filter = EXTS_FILTER_PYTHON_PACKAGES
+            exts_filter = (orig_exts_filter[0].replace('python', self.python_cmd), orig_exts_filter[1])
```

Comment:
for extensions being used with the system Python, we need to make sure we use the right Python command when doing `import` tests (which may be `python2` rather than `python`)

this is important for installing the GC3Pie bundle, for example (https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.2.eb)